lib/ews/soap.rb

```diff
@@ -64,6 +64,14 @@ module SOAP
       SOFT_DELETE = 'SoftDelete'
       MOVE_TO_DELETED_ITEMS = 'MoveToDeletedItems'
 
+      CUSTOMISABLE_HTTP_HEADERS = {
```

Comment:
from my point of view, these headers and cookies are required and very specific for EWS Affinity only - can we rename it to something like `AFFINITY_HTTP_HEADERS` according to my poor naming skill 😂